docs/platform/manager/manually-enable-elastic-ip-on-your-linode/index.md

```diff
@@ -0,0 +1,524 @@
+      {{</ file >}}
+
+1. Run the VTY shell:
+
+        sudo vtysh
+
+1. Enter configuration mode:
+
+        conf t
+
+1. Copy the contents of your template configuration file and paste them into the VTY shell:
+
+1. Tell the VTY shell that you are done entering your configurations:
+
+        end
+
+1. Write your configurations to VTY:
+
+        write
+
+1. Verify that the configurations you entered were correctly written by showing VTY's running configuration:
+
+        show running-config
+
+1. Exit out of the VTY shell:
+
+        q
+
+1. Configure the Linode's interface(s) with the Elastic IP:
+
+    > **Debian 10 & Ubuntu 18.04**
+    >
+    > Edit your Linode's `/etc/network/interfaces` file with the following entries. Replace `[ELASTIC_IP]` with the Elastic IPv4 address:
+    > {{< file >}}
+up   ip addr add [ELASTIC_IP]/24 dev eth0 label eth0:1
+down ip addr del [ELASTIC_IP]/24 dev eth0 label eth0:1
+        {{</ file >}}
+    >If you configured more than one Elastic IP on your Linode, you can add additional interface entries to your network interfaces configuration file as follows:
+
+    >{{< file >}}
+up   ip addr add [ELASTIC_IP]/24 dev eth0 label eth0:1
+down ip addr del [ELASTIC_IP]/24 dev eth0 label eth0:1
+up   ip addr add [ELASTIC_IP]_2/24 dev eth0 label eth0:2
+down ip addr del [ELASTIC_IP]_2/24 dev eth0 label eth0:2
+        {{</ file >}}
+    > **Ubuntu 20.04**
+    >
+    > Edit your Linode's `/etc/systemd/network/05-eth0.network` file by adding an `Address` entry for the Elastic IP. Replace `[ELASTIC_IP]` with the Elastic IPv4 address:
+    > {{< file >}}
+[Match]
+Name=eth0
+...
+Address=[ELASTIC_IP]/24
+        {{</ file >}}
+    >If you configured more than one Elastic IP on your Linode, you can add additional interface entries to your network interfaces configuration file as follows:
+
+    >{{< file >}}
+Address=[ELASTIC_IP]/24
+Address=[ELASTIC_IP]_2/24
+    {{</ file >}}
+    > **CentOS 8**
+    >
+    > Edit your Linode's `/etc/sysconfig/network-scripts/ifcfg-eth0` file with the following entry. Replace `[ELASTIC_IP]` with the Elastic IPv4 address:
+    > {{< file >}}
+IPADDR1=[ELASTIC_IP]
+PREFIX1="24"
+        {{</ file >}}
+    >If you configured more than one Elastic IP on your Linode, you can add additional interface entries to your network interfaces configuration file as follows:
+
+    >{{< file >}}
+IPADDR1=[ELASTIC_IP]
+PREFIX1="24"
+
+IPADDR2=[ELASTIC_IP]_2
+PREFIX2="24"
+    {{</ file >}}
+
+1. Apply the `eth0` network interface configuration:
+
+    > **Debian 10, Ubuntu 18.04 & CentOS 8**
+    >
+    > Edit your Linode's `/etc/network/interfaces` file with the following entries. Replace `[ELASTIC_IP]` with the Elastic IPv4 address:
+    >
+    >
+        sudo ifdown eth0
+        sudo ifup eth0
+    > **Ubuntu 20.04**
+    >
+    > Edit your Linode's `/etc/network/interfaces` file with the following entries. Replace `[ELASTIC_IP]` with the Elastic IPv4 address:
```

Comment:
Same as above, what am I replacing here?

Reply:
Removed that line. Should be good now.